src/onboarding/Deployment/DeploymentStepsPanel.js

```diff
@@ -55,14 +55,18 @@ function DeploymentStepsPanel({ transactionsStatus, pending, allSuccess }) {
         </h1>
 
         <div>
-          {transactionsStatus.map(({ name, status }, index) => (
-            <DeploymentStepsItem
-              key={index}
-              index={index}
-              name={name}
-              status={status}
-            />
-          ))}
+          {transactionsStatus.map(
+            ({ name, status, dateStart, gasPrice }, index) => (
+              <DeploymentStepsItem
+                key={index}
```

Comment:
It would improve the readability if the props are ordered alphabetically, it is also considered a good convention, feel free to skip this change if you think the oposite way